common/build/build-common/api-extractor-report.esm.legacyAlpha.json

```diff
@@ -0,0 +1,17 @@
+{
+	"$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+	"extends": "./api-extractor-base.esm.json",
+	"mainEntryPointFilePath": "<projectFolder>/lib/legacyAlpha.d.ts",
+	"apiReport": {
+		"reportFileName": "<unscopedPackageName>.legacy",
+		"reportVariants": ["public", "beta", "alpha"]
```

Comment:
created a new file here as they are directly referenced from the build. i think we should produce all files, eventually we can also update CODEOWNERs to exclude alpha, like we do for the default